mailbox/event/event-rabbitmq/src/test/java/org/apache/james/mailbox/events/LocalListenerRegistryTest.java

```diff
@@ -124,7 +124,7 @@ void removeListenerShouldReturnLastListenerRemovedWhenOneListener() {
 
     @Nested
     class ConcurrentTest {
-        private final Duration ONE_SECOND = Duration.ofSeconds(1);
+        private final Duration oneSecond = Duration.ofSeconds(1);
```

Comment:
could have been static instead?

Reply:
Hmmm no, cannot declare static in an inner class. Or you would need to move it outside the nested class